tst/Profiler.Tests.ps1

```diff
@@ -105,9 +105,7 @@ Describe "Trace-Script" {
         } }
         $trace = Trace-Script $sb
 
-        # get line inside foreach-object.
-        # sorting by text as lines are default sorted by Name which is unpredictable with scriptblocks
-        $inner = $trace.AllLines | Sort-Object Text | Select-Object -Last 1
+        $inner = $trace.AllLines | Where-Object Text -like '$i = $_*2; *'  | Select-Object -Last 1
```

Comment:
@fflaten the output changed, and this broke. This is imho better way, because without the comment I would have no idea which line you meant. 🙂